From 484dbffdc8443d292eb8ab0432af0409c767652a Mon Sep 17 00:00:00 2001
From: Xu Jianqun <jay.xu@rock-chips.com>
Date: Mon, 15 Feb 2016 15:30:05 +0800
Subject: [PATCH] ARM64: dts: rockchip: rk3368 reserved 128M memory for cma

Required by isp, the continuous memory region should reserve
128M for cma-heap.

Change-Id: I906fa713ee111c0a5242ffaa42416d9bd568d3a3
Signed-off-by: Xu Jianqun <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368-tb.dtsi | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368-tb.dtsi b/arch/arm64/boot/dts/rockchip/rk3368-tb.dtsi
index 43f3360f414a..38f9a4be620c 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-tb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368-tb.dtsi
@@ -50,6 +50,20 @@
 		bootargs = "console=uart,mmio32,0xff690000";
 	};
 
+	reserved-memory {
+		#address-cells = <2>;
+		#size-cells = <2>;
+		ranges;
+
+		/* global autoconfigured region for contiguous allocations */
+		linux,cma {
+			compatible = "shared-dma-pool";
+			reusable;
+			size = <0x0 0x8000000>;
+			linux,cma-default;
+		};
+	};
+
 	ion {
 		compatible = "rockchip,ion";
 		#address-cells = <1>;
-- 
2.35.3

